web/elm/src/SideBar/SideBar.elm

```diff
@@ -217,24 +233,97 @@ view model currentPipeline =
 tooltip : Model m -> Maybe Tooltip.Tooltip
 tooltip { hovered } =
     case hovered of
-        HoverState.Tooltip (SideBarTeam teamName) _ ->
+        HoverState.Tooltip (SideBarTeam _ teamName) _ ->
             Just
                 { body = Html.div Styles.tooltipBody [ Html.text teamName ]
-                , attachPosition = { direction = Tooltip.Right, alignment = Tooltip.Middle 30 }
-                , arrow = Just { size = 15, color = Colors.frame }
+                , attachPosition =
+                    { direction =
+                        Tooltip.Right (Styles.tooltipArrowSize - Styles.tooltipOffset)
+                    , alignment = Tooltip.Middle <| 2 * Styles.tooltipArrowSize
+                    }
+                , arrow = Just { size = Styles.tooltipArrowSize, color = Colors.frame }
                 }
 
-        HoverState.Tooltip (SideBarPipeline pipelineID) _ ->
+        HoverState.Tooltip (SideBarPipeline _ pipelineID) _ ->
             Just
                 { body = Html.div Styles.tooltipBody [ Html.text pipelineID.pipelineName ]
-                , attachPosition = { direction = Tooltip.Right, alignment = Tooltip.Middle 30 }
-                , arrow = Just { size = 15, color = Colors.frame }
+                , attachPosition =
+                    { direction =
+                        Tooltip.Right <|
+                            Styles.tooltipArrowSize
+                                + (Styles.starPadding * 2)
+                                + Styles.starWidth
+                                - Styles.tooltipOffset
+                    , alignment = Tooltip.Middle <| 2 * Styles.tooltipArrowSize
+                    }
+                , arrow = Just { size = Styles.tooltipArrowSize, color = Colors.frame }
                 }
 
         _ ->
             Nothing
 
 
+allPipelinesSection : Model m -> Maybe (PipelineScoped a) -> List (Html Message)
+allPipelinesSection model currentPipeline =
+    [ Html.div Styles.sectionHeader [ Html.text "all pipelines" ]
+    , Html.div [ id "all-pipelines" ]
+        (model.pipelines
+            |> RemoteData.withDefault []
+            |> List.Extra.gatherEqualsBy .teamName
+            |> List.map
+                (\( p, ps ) ->
+                    Team.team
+                        { hovered = model.hovered
+                        , pipelines = p :: ps
+                        , currentPipeline = currentPipeline
+                        , favoritedPipelines = model.favoritedPipelines
+                        , isFavoritesSection = False
+                        }
+                        { name = p.teamName
+                        , isExpanded = Set.member p.teamName model.expandedTeamsInAllPipelines
+                        }
+                        |> Views.viewTeam
+                )
+        )
+    ]
+
+
+allFavoritedPipelinesSection : Model m -> Maybe (PipelineScoped a) -> List (Html Message)
```

Comment:
```suggestion
favoritedPipelinesSection : Model m -> Maybe (PipelineScoped a) -> List (Html Message)
```
I find 'all favorited' suggests that there is some interesting subcategories of favorited that i might be concerned with